src/packages/core/src/options/server-options.ts

```diff
@@ -42,6 +43,17 @@ export type ServerConfig = {
       type: string;
       hasDefault: true;
     };
+
+    /**
+     * For memory and performance reasons ganache may respond with chunked
+     * transfer-encoding over HTTP and fragmented send over WebSockets.
+     * This option allows you to control the approximate size of each chunk.
```

Comment:
The reason it is the "approximate size":.

If a piece a data we are processing is too big to fit in the chunk we're been building we go ahead and send the chunk, even though it isn't yet full. If the piece of data we are processing is too big to fit in out chunk we send the whole piece of data as is. So the actual chunk size sent is rarely ever _exactly_ `chunkSize`.